src/components/pages/rules/flyouts/ruleNew/ruleNew.js

```diff
@@ -33,16 +47,25 @@ export class RuleNew extends LinkedComponent {
     this.state = {
```

Comment:
There is a TODO on line 41 that can be removed.

Reply:
fixed